samples/bookinfo/build_push_update_images.sh

```diff
@@ -30,16 +31,16 @@ src/build-services.sh "${VERSION}"
 #get all the new image names and tags
 for v in ${VERSION} "latest"
 do
-  IMAGES+=$(docker images -f reference=istio/examples-bookinfo*:"$v" --format "{{.Repository}}:$v")
+  IMAGES+=$(docker images -f reference=$HUB/examples-bookinfo*:"$v" --format "{{.Repository}}:$v")
```

Comment:
Could you please use `${VARIABLE}` instead of `$VARIABLE`?

Reply:
done